kola/tests/coretest/core.go

```diff
@@ -275,6 +296,49 @@ func servicesActive(units []string) error {
 	return nil
 }
 
+func TestServicesDisabledRHCOS() error {
+	inactiveErr := servicesInactive(offServices)
```

Comment:
No need to use separate error variables as you're not returning both, just re-use a single `err` variable.